index.html

```diff
@@ -65,13 +70,14 @@ <h2>Connect</h2>
     </main>
 
     <footer>
-        <hr>
+        <!-- <hr>
```

Comment:
If you no longer need a bit of code, it's usually best to just delete it, to keep things from getting cluttered. If you want to bring it back, you can always find it in the git history